python/tvm/driver/tvmc/model.py

```diff
@@ -53,6 +53,7 @@
 from tvm import relay
 from tvm.contrib import utils
 from tvm.relay.backend.executor_factory import GraphExecutorFactoryModule
+from tvm.micro import export_model_library_format
```

Comment:
i'm wondering if it makes sense to keep `USE_MICRO`. i may propose removing it on Discuss. Currently, `USE_MICRO` means to build the utvm_rpc_common library from the CRT into `libtvm_runtime.so`. I don't know that adding this is a huge burden, but on the other hand I do want to be sensitive to people trying to compile for smaller runtime environments.

Reply:
@leandron  fwiw if `USE_MICRO OFF` an error is already generated:
```    
<snip>
raise ImportError("micro tvm is not enabled. Set USE_MICRO to ON in config.cmake")
ImportError: micro tvm is not enabled. Set USE_MICRO to ON in config.cmake
```

Reply:
Ack. I sent https://github.com/tlc-pack/tlcpack/pull/59 to make sure the package we generate also builds with `USE_MICRO ON`, so that it is enabled by default for users wanting to use the feature implemented in this PR.